index.js

```diff
@@ -32,7 +32,21 @@ const saveAs = (uri, filename) => {
  * @param  {string} type
  * @param  {object} options=null
  */
-const exportComponent = (node, fileName, backgroundColor, type, options) => {
+
+const getPDF = (type, pdfOrientation) => {
+    if(pdfOrientation === 'l') {
+        return new JsPDF('l', 'mm', [canvas.width, canvas.height])
+    } else if(pdfOrientation === 'p') {
+        return new JsPDF('p', 'mm', [canvas.height, canvas.width]);
+    } else {
+        return canvas.width > canvas.height
+        ? new JsPDF('l', 'mm', [canvas.width, canvas.height])
+        : new JsPDF('p', 'mm', [canvas.height, canvas.width]);
+    }
+    
+}
+
+const exportComponent = (node, fileName, backgroundColor, type, options, pdfOrientation = '') => {
```

Comment:
Hi, can we define options object for pdf which will have x, y and orientation. With that we will tackle as well.

https://github.com/salman-monetate/react-component-export-image/issues/19